Casks/rewind.rb

```diff
@@ -0,0 +1,43 @@
+cask "rewind" do
+  version "1.0041,20230419"
+  sha256 :no_check
+
+  url "https://download.rewind.ai/Rewind.dmg"
+  name "Rewind"
+  desc "Record and search your screen and audio"
+  homepage "https://www.rewind.ai/"
+
+  livecheck do
+    url "https://updates.rewind.ai/appcasts/main.xml"
+    strategy :xml do |xml|
+      # Throttle updates to one every 3 days.
+      next version if DateTime.parse(version.csv.second) + 3 > Date.today
+
+      item = xml.get_elements("//item")[0]
+      version = item.elements["sparkle:shortVersionString"].text
+      date = DateTime.parse(item.elements["pubDate"].text).strftime("%Y%m%d")
+
+      "#{version},#{date}"
```

Comment:
```suggestion
    strategy :sparkle do |item|
      # Throttle updates to one every 3 days.
      next version if DateTime.parse(version.csv.second) + 3 > Date.today

      "#{item.short_version},#{item.pub_date.strftime("%Y%m%d")}"
```

As Markus mentioned, we can use the `Sparkle` strategy here. It takes care of most of the custom logic (e.g., getting the first item, pulling out the `shortVersionString` text, and parsing the `pubDate` into a `Time`/`DateTime` object), so the `strategy` block ends up pretty simple.

Reply:
Thank you! This is super helpful. At some point, I'll update the documentation in the Caskbook, so other people don't try to reinvent the wheel in the future. :)